modules/cluster-logging-deploy-storage-considerations.adoc

```diff
@@ -69,19 +63,11 @@ calculate per-application throughput and disk space as follows:
 
 Fluentd ships any logs from *systemd journal* and **/var/log/containers/*.log** to Elasticsearch.
 
-Therefore, consider how much data you need in advance and that you are
-aggregating application log data. Some Elasticsearch users have found that it
-is necessary to keep absolute storage consumption around 50% and below 70% at all times. This
-helps to avoid Elasticsearch becoming unresponsive during large merge
-operations.
+Therefore, consider how much data you need in advance and that you are aggregating application log data. Some Elasticsearch users have found that it is necessary to keep absolute storage consumption around 50% and below 70% at all times. This helps to avoid Elasticsearch becoming unresponsive during large merge operations.
 
-By default, at 85% Elasticsearch stops allocating new data to the node, at 90% Elasticsearch attempts to relocate
-existing shards from that node to other nodes if possible. But if no nodes have free capacity below 85%, Elasticsearch effectively rejects creating new indices
-and becomes RED.
+By default, at 85% Elasticsearch stops allocating new data to the node, at 90% Elasticsearch attempts to relocate existing shards from that node to other nodes if possible. But if no nodes have free capacity below 85%, Elasticsearch effectively rejects creating new indices and becomes RED.
 
 [NOTE]
 ====
-These low and high watermark values are Elasticsearch defaults in the current release. You can modify these values,
-but you also must apply any modifications to the alerts also. The alerts are based
-on these defaults.
+These low and high watermark values are Elasticsearch defaults in the current release. The alerts are based on these defaults.
```

Comment:
Can we keep the verbiage about how its possible to change these default values in Elasticsearch? Because it is possible to.. however we can't change the value for the alerts.

"These low and high watermark values are the Elasticsearch defaults in the current release. You can modify these values, however the alerts are based on these defaults and are not able to be updated."

Reply:
Thanks, @ewolinetz. Updated. Please take another look.